app/models/message.rb

```diff
@@ -5,7 +5,8 @@ class Message < ApplicationRecord
 
   default_scope { order(created_at: :desc) }
 
-  multisearchable against: :text, if: :reply?
+  multisearchable against: :text, if: :reply?,
+                  additional_attributes: ->(message) { { organization_id: message.request.organization_id } }
 
```

Comment:
should we delegate "organization" from "message" to "request" as a message always belongs to one request?